src/proxy-ui-api/frontend/src/components/layout/AppToolbar.vue

```diff
@@ -80,7 +80,17 @@ export default Vue.extend({
   },
   methods: {
     home(): void {
-      this.$router.replace({ name: RouteName.Clients });
+      this.$router
+        .replace({
+          name: this.$store.getters.firstAllowedTab.to.name,
+        })
+        .catch((err) => {
```

Comment:
The fix in itself is ok. I was thinking whether we should not completely swallow the expection but instead add somekind of console.log notice etc. I wonder if there could now be an endless routing loop and we wouldn't know why the app would fail?

Reply:
I added a console.log there. Now there is some clue if something weird starts happening.